proposal-template.md

```diff
@@ -0,0 +1,31 @@
+# Proposal name
+
+**Author**: Name here
+
+**Status**: Draft
+
+**Last updated**: Date here
```

Comment:
Can you include a created date, as well?

Reply:
Done